packages/ember-routing-htmlbars/tests/helpers/closure_action_test.js

```diff
@@ -583,3 +584,38 @@ QUnit.test('objects that define INVOKE can be casted to actions', function(asser
 });
 
 }
+
+if (isEnabled('ember-improved-instrumentation')) {
+  QUnit.test('action should fire interaction event', function(assert) {
+    assert.expect(2);
+
+    subscribe('interaction.ember-action', {
+      before() {
+        console.log(arguments);
+        assert.ok(true, 'instrumentation subscriber was called');
+      },
+      after() {
+      }
+    });
+
+    innerComponent = EmberComponent.extend({
+      fireAction() {
+        this.attrs.submit();
+      }
+    }).create();
+
+    outerComponent = EmberComponent.extend({
+      layout: compile('{{view innerComponent submit=(action outerSubmit)}}'),
```

Comment:
I think you would need to introduce the `owner` stuff that is in the local-look-test. Basically you need a means of registering.